app/src/main/java/de/danoeh/antennapod/fragment/QuickFeedDiscoveryFragment.java

```diff
@@ -138,7 +164,11 @@ private void loadToplist() {
             return;
         }
 
-        disposable = loader.loadToplist(countryCode, NUM_SUGGESTIONS)
+        int numRequesting = NUM_SUGGESTIONS + subscribedFeeds.size();
+        if (numRequesting > ITUNES_LIMIT) {
+            numRequesting = ITUNES_LIMIT;
+        }
+        disposable = loader.loadToplist(countryCode, numRequesting)
```

Comment:
I would just always load 25 items, same as on the expanded page. That way we don't create additional traffic on the iTunes API and risk getting blocked. 25 already is more than we display here, so there is some room for users to subscribe to stuff before the list looks empty.